### PR TITLE
[#744] Profile page hotfix

### PR DESCRIPTION
**Pull request by mauriel133**
This fixes a minor bug where some authors are not listed for a resource item in the profile page.
Also includes minor style changes.

### CODE REVIEW

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

File: theme/static/css/hydroshare_core.css [hunk 1]
```diff
@@ -2200,6 +2200,7 @@ div.affiliation {
 .activity-block {
     margin-right: 0;
     margin-left: 0;
+	color: #757575;
```
Comment (ironfroggy) [inline theme/static/css/hydroshare_core.css]: please do not mix tabs and spaces.

File: theme/templates/accounts/profile.html [hunk 2]
```diff
@@ -321,23 +321,22 @@ <h3 class="panel-title"><span class="glyphicon glyphicon-phone-alt">&nbsp;</span
                                     <div class="activity-block row">
                                         <div class="activity-info col-sm-12">
                                             <span style="font-size:16px;"><a href="{{ res.get_absolute_url }}">{{ res.metadata.title }}</a></span>
-                                            <h5 style="margin:0" class="text-muted">Created: <strong>{{ res.created }}</strong></h5>
-                                            <h5 style="margin:0" class="text-muted">Authors:<strong>
+                                            <h5 style="margin:0; color:#757575;">Created: {{ res.created }}</h5>
+                                            <h5 style="margin:0; color:#757575;">Authors:
```
Comment (ironfroggy) [inline theme/templates/accounts/profile.html]: these styles belong in the stylesheet.

**Comment by hyi** [comment]
@mauriel133 I am waiting for you to make a couple of quick fixes to address @ironfroggy 's comments, then I'll merge it into develop branch and get ready to deploy release 1.6.2 on www.

**Comment by mauriel133** [comment]
Will push the changes before the meeting.

**Comment by mauriel133** [comment]
I have addressed the issues above now.

**Comment by ironfroggy** [comment]
+1

**Pull request merged**